cpp/src/indexing/construction/detail/phase_1.cuh

```diff
@@ -224,10 +225,10 @@ reverse_tree_levels(rmm::device_vector<uint32_t> const &quad_keys_in,
   }
 
   // Shrink vectors' underlying device allocations to reduce peak memory usage
-  quad_keys.shrink_to_fit();
-  quad_point_count.shrink_to_fit();
-  quad_child_count.shrink_to_fit();
-  quad_levels.shrink_to_fit();
+  quad_keys.shrink_to_fit(stream);
```

Comment:
It doesn't seem like these vectors were ever resized, so this shrink to fit isn't going to do anything.